pkg/manager/manager.go

```diff
@@ -0,0 +1,84 @@
+package manager
+
+import (
+	"os"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	dcv1alpha1 "github.com/dominodatalab/distributed-compute-operator/api/v1alpha1"
+	"github.com/dominodatalab/distributed-compute-operator/controllers"
+	//+kubebuilder:scaffold:imports
+)
+
+var (
+	scheme   = runtime.NewScheme()
+	setupLog = ctrl.Log.WithName("setup")
+)
+
+func Start(cfg *Config) error {
+	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&cfg.ZapOptions)))
+
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Scheme:                 scheme,
+		MetricsBindAddress:     cfg.MetricsAddr,
+		Port:                   9443,
+		HealthProbeBindAddress: cfg.HealthProbeAddr,
+		LeaderElection:         cfg.EnableLeaderElection,
+		LeaderElectionID:       "a846cbf2.dominodatalab.com",
+		Namespace:              cfg.Namespace,
+	})
+	if err != nil {
+		setupLog.Error(err, "unable to start manager")
+		return err
+	}
+
+	if err = (&controllers.RayClusterReconciler{
+		Client: mgr.GetClient(),
+		Log:    ctrl.Log.WithName("controllers").WithName("RayCluster"),
+		Scheme: mgr.GetScheme(),
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "RayCluster")
+		return err
+	}
+
+	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
```

Comment:
Why is this an env variable rather than a CLI arg?

Reply:
Great question. I don't want users to EVER disable webhooks but I wanted to provide developers a way to disable the webhooks so they could execute the binary directly on their workstation with a debugger if necessary. I think there's probably something more clever that I can do so devs can run the entire project from their workstation but I backlogged that and added this a quick fix.